README.md

```diff
@@ -170,10 +170,10 @@ Offered under the [Apache 2 license][license].
 [Buf Studio]: https://buf.build/studio
 [Getting Started]: https://connectrpc.com/docs/go/getting-started
 [blog]: https://buf.build/blog/connect-a-better-grpc
-[connect-crosstest]: https://github.com/bufbuild/connect-crosstest
+[connect-conformance]: https://github.com/connectrpc/conformance
```

Comment:
@akshayjshah do you think this should be `conformance` or `connect-conformance`?

Reply:
I'd probably vote `conformance`, but only very slightly / don't have a strong opinion - `conformance` is sort of weird, but it is consistent.